mne/io/tests/test_raw.py

```diff
@@ -166,10 +166,14 @@ def _test_raw_reader(
             fnirs, atol = "hbo", 1e-10
         elif "hbr" in raw:
             fnirs, atol = "hbr", 1e-10
-        else:
-            assert "fnirs_cw_amplitude" in raw, "New channel type necessary?"
+        elif "fnirs_cw_amplitude" in raw:
             fnirs, atol = "fnirs_cw_amplitude", 1e-10
-        picks = pick_types(other_raw.info, meg=meg, eeg=eeg, fnirs=fnirs)
+        else:
+            # e.g., https://github.com/mne-tools/mne-python/pull/11432/files
```

Comment:
What happened here? Don't we already support seeg channel types in other file formats?

Reply:
If we have any such files, we must not currently be doing `_test_raw_reader` on them (which is not good!)

Reply:
Or maybe the channel type is not set to 'seeg' while reading those files? The channels were set 'seeg' after the raw is constructed. I do that when the seeg recordings are stored in edf.